src/BigQuery/ValueMapper.php

```diff
@@ -346,7 +346,12 @@ private function timestampFromBigQuery($value)
             }
         }
 
-        list($unixTimestamp, $microSeconds) = explode('.', $value);
+        $parts = explode('.', $value);
```

Comment:
@dwsupplee we discussed this offline. It was causing failures in the BigQuery suite and you mentioned you were looking into it. If you've got a fix which renders this irrelevant let me know and I'll revert this.